index.html

```diff
@@ -54,7 +54,240 @@ <h2 class="contact-title">LET'S CONNECT</h2>
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <div class="social-media-bottom">
+            <a class="contact-link" href="#">LET'S CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="https://twitter.com/Ritikarawat22"
+                  ><img src="images/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/rawatritika/"
+                  ><img
+                    src="images/Linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="images/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/ritikarawat220"
+                  ><img src="./images/Github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+            <button class="btn-default" type="button">Get my resume</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-dow.svg" alt="" />
+                </button>
+              </div>
+              <!-- <div class="separator-line"></div> -->
+              <ul class="language-list">
+                <li>
+                  <img src="./javascript.svg" alt="JavaScript" />
+                  JavaScript
+                </li>
+                <li>
+                  <img src="./html.svg" alt="HTML" />
+                  HTML
+                </li>
+                <li>
+                  <img src="./css.svg" alt="CSS" />
+                  CSS
+                </li>
+              </ul>
+            </li>
+
+            <!-- /languages -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Frameworks</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+            </li>
+            <!-- /Frameworks -->
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Skills</h3>
+                <button class="btn-chev" type="button">
+                  <img src="./chev-right.svg" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
```

Comment:
- [ ] You should stick to the design as much as possible ie: including color, background, and spacing. Please adjust the spacing for the hx tag also.
### Your design
![his](https://user-images.githubusercontent.com/81727988/212310832-cd36ddcf-77c3-47c5-8c0b-613eb8826ea6.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/212310819-6ed921f7-06f8-4912-a257-87885a9fe86a.jpg)